app-web/plugins/gatsby-source-github-all/utils/helpers.js

```diff
@@ -87,15 +87,18 @@ const getClosestResourceType = resourceType => {
 /**
  * returns the closest persona from the array of personas based on the
  * uncontrolled persona (given to us by contributors)
- * @param {String} persona the persona provided by a specific piece of content
+ * @param {Array} personas the personas provided by a specific piece of content
  * @param {Array} personas the valid personas list
  */
-const getClosestPersona = (persona, personas) => {
+const getClosestPersona = (personaList, personas) => {
   const RATING_THRESHOLD = 0.5; // rating is between 0 - 1, we only want a match if it's greater than half.
   // if its blank don't bother checking closeness
-  if (persona === '') return '';
```

Comment:
if you are keep it for backward support, might be good to still include the check here for now.

Reply:
there is still backwards support for persona configuration wise. Code wise, a persona is mapped to personas as an array. get closest persona was converted to map over the personas array `personaList`

Reply:
I see. Good to go!

Reply:
doooppeee :)